draft-ietf-quic-recovery.md

```diff
@@ -228,6 +228,23 @@ QUIC endpoints measure the delay incurred between when a packet is received and
 when the corresponding acknowledgment is sent, allowing a peer to maintain a
 more accurate round-trip time estimate (see Section 13.2 of {{QUIC-TRANSPORT}}).
 
+### Probe Timeout Replaces RTO and TLP
+
+QUIC uses a probe timeout (see {{pto}}), with a timer based on TCP's
+RTO timeout.  QUIC PTO includes the peer's max_ack_delay in the calculation,
+instead of relying upon a fixed minimum RTO.  Unlike TCP's RTO, which collapses
+the congestion window upon expiry, QUIC does not change the congestion window
+and instead allows sending probe packets whenever the timer expires.  This is
+similar to TCP with F-RTO, but it does allow more packets to be sent when the
+congestion window was not fully utilized when the probe timeout expires. Though
+this is slightly more aggressive than TCP RTO, it's less aggressive than if the
+connection was not application limited.
```

Comment:
I'm not sure I understand this last sentence... how is app-limited relevant here?

Reply:
ping

Reply:
If you weren't app-limited, then you wouldn't have CWND free to send more packets after the PTO expires.